engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -142,12 +142,13 @@ object WorkflowActor {
             jobStoreActor: ActorRef,
             subWorkflowStoreActor: ActorRef,
             callCacheReadActor: ActorRef,
+            callCacheWriteActor: ActorRef,
             dockerHashActor: ActorRef,
             jobTokenDispenserActor: ActorRef,
             backendSingletonCollection: BackendSingletonCollection,
             serverMode: Boolean): Props = {
     Props(new WorkflowActor(workflowId, startMode, wdlSource, conf, ioActor, serviceRegistryActor, workflowLogCopyRouter,
-      jobStoreActor, subWorkflowStoreActor, callCacheReadActor, dockerHashActor, jobTokenDispenserActor, backendSingletonCollection, serverMode)).withDispatcher(EngineDispatcher)
+      jobStoreActor, subWorkflowStoreActor, callCacheReadActor, callCacheWriteActor, dockerHashActor, jobTokenDispenserActor, backendSingletonCollection, serverMode)).withDispatcher(EngineDispatcher)
```

Comment:
In other cases where we have a  long list of (essentially untyped) `ActorRef` arguments, I find it pretty much necessary to add the param names to avoid mixing up the order, e.g.
```
Props(new WorkflowActor(
 ...
  jobStoreActor = jobStoreActor,
  subWorkflowStoreActor = subWorkflowStoreActor,
  ...
```